### DISCUSSION TITLE
scene.on('click', (ev) => {})   方法中返回的参数中，哪些参数，我可以确定这个位置属于哪个省份？

### DISCUSSION THREAD

**Discussion by DreamNO1** [antvis/L7#1858] [Q&A] [answered]
1.问题
 scene.on('click', (ev) => {})   方法中返回的参数中，哪些参数，我可以确定这个位置属于哪个省份？  谢谢
    因为从我目前来看，好像只有经纬度可以确定属于哪个省份，不知道有没有其他的参数可以确定数据那个省份，谢谢

**Comment by lzxue** [comment, marked as answer]
不能，经纬转省份需要自己计算，或者调用 3 方 API
